### PR TITLE
Add extra requirments.txt for converting gptq

### PR DESCRIPTION
**Pull request by wgzintel**

### CODE REVIEW

File: llm_bench/python/extra/requirements_gptq_conversion.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+auto-gptq==0.5.1 # for gptq
```
Comment (peterchen-intel) [inline llm_bench/python/extra/requirements_gptq_conversion.txt:1]:
```suggestion
auto-gptq>=0.5.0 # for gptq
```
Can 0.5.0 work?

**Comment by peterchen-intel** [comment]
Got following error for chatglm2-6b-gptq and chatglm3-6b-gptq

```
$python benchmark.py -m ./chatglm2-6b-gptq/pytorch/dldt/GPTQ_INT4-FP16
...
[ INFO ] Input token size:9, max_output_token_size:512
[ ERROR ] An exception occurred
[ INFO ] Traceback (most recent call last):
  File "./openvino.genai/llm_bench/python/benchmark.py", line 469, in main
    iter_data_list, pretrain_time = CASE_TO_BENCH[model_args['use_case']](model_path, framework, args.device, model_args, args.num_iters)
  File "./openvino.genai/llm_bench/python/benchmark.py", line 165, in run_text_generation_benchmark
    run_text_generation(input_text, num, model, tokenizer, args, iter_data_list, prompt_idx, bench_hook)
  File "./openvino.genai/llm_bench/python/benchmark.py", line 95, in run_text_generation
    result = model.generate(**input_data, max_new_tokens=int(max_output_token_size), num_beams=args['num_beams'], use_cache=True)
  File "./weekly_test/ww47/py310_d9b5e753/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "./weekly_test/ww47/py310_d9b5e753/lib/python3.10/site-packages/transformers/generation/utils.py", line 1673, in generate
    return self.greedy_search(
  File "./openvino.genai/llm_bench/python/utils/hook_transformers.py", line 206, in new_greedy_search
    outputs = self(
  File "./weekly_test/ww47/py310_d9b5e753/lib/python3.10/site-packages/optimum/modeling_base.py", line 90, in __call__
    return self.forward(*args, **kwargs)
  File "./weekly_test/ww47/py310_d9b5e753/lib/python3.10/site-packages/optimum/intel/openvino/modeling_decoder.py", line 361, in forward
    inputs[input_name] = Tensor(model_inputs.get_element_type(), shape.get_shape())
RuntimeError: Check 'min_val == max_val' failed at src/core/src/partial_shape.cpp:131:
get_shape() must be called on a static shape
```

File: llm_bench/python/README.md [hunk 2]
```diff
@@ -10,6 +10,15 @@ source python-env/bin/activate
...
 pip install -r requirements.txt
 ```
+---
+***Requirments for GPTQ models***<br />
+To convert the GPTQ model, execute the following commands after installing above requirements.txt
+``` bash
+pip uninstall auto-gptq
+pip install -r extra/requirements_gptq_conversion.txt
+```
+---
```
Comment (peterchen-intel) [inline llm_bench/python/README.md:20]:
```suggestion
***Warning***<br />
Create another Python virtual environment to benchmark.py the converted models and do not install extra/requirements_gptq_conversion.txt
---
```

File: llm_bench/python/README.md
Comment (skuros) [inline llm_bench/python/README.md:1]: Please change to:
`pip install -r requirements.txt`

**Pull request closed without merging**